verify-cborld-presentation/src/App.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2019 - MATTR Limited
+ * All rights reserved
+ * Confidential and proprietary
+ */
+
+import React, { useState } from "react";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import styled from "styled-components";
+
+import { HomePage, ScanPage, ResultPage, QrPage } from "./pages";
+
+export const App = (): React.ReactElement => {
+  const [qrCode, setQrCode] = useState<string>();
+
+  const onQrCodeChange = (scannedQrCode: string): void => {
+    setQrCode(scannedQrCode);
+  };
+
+  return (
+    <Container>
+      <Nav>
+        <Logo src="./static/mattr-logo.svg" />
+      </Nav>
+      <Content>
+        <Router>
```

Comment:
Would be nice to extract routing into a separate component, 

```
<Route path="/result" component={ResultPage} />
...
```

Reply:
Makes sense thanks, have moved to own component